### PR TITLE
Add aarch64 support

### LINKED ISSUE #4: CI for AArch64 architecture

### PR DESCRIPTION
**Pull request by Seia-Soto**
Implemented ARM build via `--runtime-file` option to fix #4

* Squash merge recommended.

### CODE REVIEW

**Comment by Seia-Soto** [comment]
Screenshot on ARM virtual machine (Apple M1 Parallels - Ubuntu Focal)

image

File: build [hunk 1]
```diff
@@ -7,21 +7,22 @@ build
 Scripts which packages AWS CLI v2 into an AppImage binary.
 
 Usage:
-  ./build <version>
+  ./build <version> [arch]
```
Comment (simnalamburt) [inline build:10]:
```suggestion
  ./build <version> <arch>
```

File: build [hunk 2]
```diff
@@ -7,21 +7,22 @@ build
...
   ./build [-h|--help]
 
   <version>     Version of AWS CLI
+  [arch]        Architecture to build, default to your arch
```
Comment (simnalamburt) [inline build:14]:
```suggestion
  <arch>        Architecture to build, default to your arch
```

File: build [hunk 3]
```diff
@@ -32,23 +33,26 @@ if [[ "${1}" == '-h' || "${1}" == '--help' ]]; then
 fi
 
 VERSION="${1}"
+ARCH="${2:-$(uname -m)}"
 
+echo "AWS CLI v2, in a single file - ${VERSION} ${ARCH}"
 
+if [[ $ARCH != 'x86_64' && $ARCH != 'aarch64' ]]; then
```
Comment (simnalamburt) [inline build:40]:
```suggestion
if [[ "$ARCH" != 'x86_64' && "$ARCH" != 'aarch64' ]]; then
```

File: build [hunk 4]
```diff
@@ -32,23 +33,26 @@ if [[ "${1}" == '-h' || "${1}" == '--help' ]]; then
...
+  echo Unsupported architecture specified! Arch should be x86_64 or aarch64.
```
Comment (simnalamburt) [inline build:41]:
```suggestion
  echo 'Unsupported architecture specified! Arch should be x86_64 or aarch64.'
```

File: build [hunk 5]
```diff
@@ -32,23 +33,26 @@ if [[ "${1}" == '-h' || "${1}" == '--help' ]]; then
...
+  exit
```
Comment (simnalamburt) [inline build:42]:
```suggestion
  exit 1
```

File: build [hunk 6]
```diff
@@ -7,21 +7,22 @@ build
...
-  ./build 2.1.14
+  ./build 2.0.0 x86_64
+  ./build 2.1.14 aarch64
   ./build -h
   ./build --help
 EOF
 }
 
-if (( $# != 1 )); then
+if [ $# -eq 0 ]; then
```
Comment (simnalamburt) [inline build:25]:
```suggestion
if (( $# == 0 )); then
```

**Review by simnalamburt — APPROVED** [review]

**Pull request merged**